src/forms/factory/createUrlSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { roomUrlSchema } from "forms/roomUrlSchema";
+
+export interface CreateUrlSchemaOptions {
+  name: string;
+  required?: boolean;
+}
+
+export const createUrlSchema = ({ name, required }: CreateUrlSchemaOptions) => {
+  let schema = roomUrlSchema;
+
+  if (required) {
+    schema = schema.required(`${name} is required`);
+  }
+
+  return schema;
+};
```

Comment:
[NOTE]

Looks like an overkill to provide a factory for simply not writing `Yup.string().required()` and `Yup.string()` as opposed to someting like `createNameSchema()` which needs to deal with more complex parameter interaction.

That being said, not really an issue to have this. Maybe it should grow in the future.

Reply:
leaving it as it is for now. maybe we'll have more means to use this schema in the near future